mis_builder/static/src/js/mis_builder_widget.js

```diff
@@ -0,0 +1,121 @@
+odoo.define('mis_builder.widget', function(require) {
+"use strict";
+
+var AbstractField = require('web.AbstractField');
+var core = require('web.core');
+var registry = require('web.field_registry');
+
+var _t = core._t;
+
+var MisBuilderWidget = AbstractField.extend({
+    events: _.extend({}, AbstractField.prototype.events, {
+        'click .mis_builder_drilldown': 'drilldown',
+        'click .oe_mis_builder_print': 'print',
+        'click .oe_mis_builder_export': 'export',
+        'click .oe_mis_builder_settings': 'display_settings',
+        'click .oe_mis_builder_generate_content': 'generate_content',
+    }),
+    description: "",
+
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+
+    /**
+     * @override
+     * it should always be displayed, whatever its value
+     */
+    isSet: function () {
+        return true;
+    },
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * @override
+     * @private
+     */
+    _render: function () {
+        var self = this;
+        var context = self.getParent().state.context;
+        var rec_id = this.recordData.id || self.getParent().state.context.active_id;
```

Comment:
Here we must use the field value. Not sure active_id makes sense (TBC).

Reply:
The problem I had is when you add to the dashboard. It is in those cases where I needed. Because in the dashboard the record is not loaded. This has changed in the definition of the dashboard from v10, as far as I can tell.